docs/photosynthesis_read_dat.md

```diff
@@ -0,0 +1,34 @@
+## Read .DAT Fluorescence Image Files
+
+Reads image into numpy ndarray and reshapes the frames into a datacube before identifying and extracting `fdark`, `fmin`, and `fmax` as separate numpy ndarrays. 
+
+**plantcv.photosynthesis.read_dat**(*filename*)
+
+**returns** fdark, fmin, fmax
+
+- **Parameters:**
+    - filename - image file to be read (possibly including a path)
+    
+- **Context:**
+    - Reads in file to be processed and does so using the metadata contained within a corresponding .INF file
+- **Notes:**
+    - This function assumes a specific pattern between .DAT image files and their corresponding .INF file. 
+    We assume that for every image file `xx_PSD_xxx.DAT` there will be a corresponding metadata file with the same path
+    named `xx_HDR_xxx.INF` 
+- **Example use:**
+    - [Use In PSII Tutorial](psII_tutorial.md) ****Needs updating**** 
```

Comment:
"Needs updating" note still relevant?

Reply:
Yes. Once we do another version release, I'll be able to open a PR for the updated tutorial. I've got a branch started right now with the new data and workflow!

Reply:
Oh wait, Binder needs updating now. This doc page is actually updated now!